qa-niaid.planx-pla.net/portal/gitops.json

```diff
@@ -26,7 +26,7 @@
           "2. Your DAR will be reviewed by the NIAID Clinical Trials Data Access Committee. ",
           "3. Upon approval of the DAR, the primary data requestor and their institution will be notified and required to agree to and sign a NIAID Data Use Agreement (DUA) online via DocuSign that outlines the terms of the use of the data.",
           "4. Upon NIAID’s receipt of the NIAID DUA, data requestors will be notified on how to download the approved data.",
-          "5. Please note you can review a blank <a href='https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAID_DAR.pdf'>DAR</a> and <a href='https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAIDDUA2021Accessclinicaldata@NIAID.pdf'>DUA</a> prior to viewing and requesting the available data, but the request must be done using a DAR online form found on the data platform, as outlined above.",
+          "5. Please note you can review a blank <a href='https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAID_DAR.pdf'>DAR</a> and <a href='https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAIDDUA2024Accessclinicaldata@NIAID.pdf'>DUA</a> prior to viewing and requesting the available data, but the request must be done using a DAR online form found on the data platform, as outlined above.",
```

Comment:
we should rename it `NIAIDDUAAccessclinicaldata@NIAID.pdf` without a date and remove unnecessary files. Github already maintains historical versions, it should be enough if we ever need to check the old file